File: src/main/java/x404/kouldyeri/model/Livreur.java
package x404.kouldyeri.model;

public class Livreur {
    private int idLivreur;
    private String nom;
    private String prenom;
    private String telephone;
    private boolean disponibilite;

    public Livreur() {}

    public Livreur(String nom, String prenom, String telephone, boolean disponibilite) {
        this.nom = nom;
        this.prenom = prenom;
        this.telephone = telephone;
        this.disponibilite = disponibilite;
    }

    public Livreur(int idLivreur, String nom, String prenom, String telephone, boolean disponibilite) {
        this.idLivreur = idLivreur;
        this.nom = nom;
        this.prenom = prenom;
        this.telephone = telephone;
        this.disponibilite = disponibilite;
    }

    public int getIdLivreur() { return idLivreur; }
    public void setIdLivreur(int idLivreur) { this.idLivreur = idLivreur; }

    public String getNom() { return nom; }
    public void setNom(String nom) { this.nom = nom; }

    public String getPrenom() { return prenom; }
    public void setPrenom(String prenom) { this.prenom = prenom; }

    public String getTelephone() { return telephone; }
    public void setTelephone(String telephone) { this.telephone = telephone; }

    public boolean isDisponibilite() { return disponibilite; }
    public void setDisponibilite(boolean disponibilite) { this.disponibilite = disponibilite; }

    // utile pour ComboBox dans Livraison
    @Override
    public String toString() {
        return idLivreur + " - " + nom + " " + prenom;
    }
}

File: src/Entity/Livreur.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\LivreurRepository;

#[ORM\Entity(repositoryClass: LivreurRepository::class)]
#[ORM\Table(name: 'livreur')]
class Livreur
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(name: 'id_livreur', type: 'integer')]
    private ?int $idLivreur = null;

    #[ORM\Column(name: 'nom', type: 'string', length: 100)]
    private ?string $nom = null;

    #[ORM\Column(name: 'prenom', type: 'string', length: 100)]
    private ?string $prenom = null;

    #[ORM\Column(name: 'telephone', type: 'string', length: 20)]
    private ?string $telephone = null;

    #[ORM\Column(name: 'disponibilite', type: 'boolean', options: ['default' => true])]
    private ?bool $disponibilite = true;

    // Getters et Setters
    public function getIdLivreur(): ?int { return $this->idLivreur; }
    public function getNom(): ?string { return $this->nom; }
    public function setNom(string $nom): self { $this->nom = $nom; return $this; }
    public function getPrenom(): ?string { return $this->prenom; }
    public function setPrenom(string $prenom): self { $this->prenom = $prenom; return $this; }
    public function getTelephone(): ?string { return $this->telephone; }
    public function setTelephone(string $telephone): self { $this->telephone = $telephone; return $this; }
    public function getDisponibilite(): ?bool { return $this->disponibilite; }
    public function setDisponibilite(bool $disponibilite): self { $this->disponibilite = $disponibilite; return $this; }
}